provider/aci.go

```diff
@@ -680,6 +680,11 @@ func (p *ACIProvider) CreatePod(ctx context.Context, pod *v1.Pod) error {
 	containerGroup.RestartPolicy = aci.ContainerGroupRestartPolicy(pod.Spec.RestartPolicy)
 	containerGroup.ContainerGroupProperties.OsType = aci.OperatingSystemTypes(p.operatingSystem)
 
+	// get initContainers
```

Comment:
you don't have to add the functionality in the v1 code (before SDK). Please remove all new func

Reply:
removed from old code